appveyor.yml

```diff
@@ -35,7 +35,7 @@ install:
 
     SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%
 
-    pip install numpy pytest pytest-cov mock
+    pip install numpy pytest pytest-cov mock "pyarrow>=0.11,<0.12" pandas
```

Comment:
This is odd. You install pyarrow 0.12 under Linux, and also the current master requires pyarrow 0.12+.

Reply:
This was an artefact from rebasing. This branch is older than the pyarrow update; fixed.